initial refactoring (package names and semver)

noahgorstein opened the pull request:
Purpose of this PR is to change the group id we publish our jars under and also to follow Semantic Versioning. I've also refactored the package names to remove complexible as we are no longer complexible :D.

snowell commented on stardog-spring-batch/build.gradle:
```diff
@@ -36,9 +36,9 @@ dependencies {
-	compile ('com.stardog:stardog-spring:7.2.1')
-	testCompile ('com.complexible.stardog:server:7.2.1') {
+	compile ('com.complexible.stardog:client-http:7.3.2')
+	compile ('com.stardog.ext:stardog-spring:1.0.0')
```
I think you can use $version here to make it easier

snowell commented on stardog-spring-batch/build.gradle:
```diff
@@ -36,9 +36,9 @@ dependencies {
-	testCompile ('com.complexible.stardog:server:7.2.1') {
+	compile ('com.complexible.stardog:client-http:7.3.2')
+	compile ('com.stardog.ext:stardog-spring:1.0.0')
+	testCompile ('com.complexible.stardog:server:7.3.2') {
```
We can/should keep this as a variable in the base build.gradle so we only have to edit it in one place. The docs could potentially pull from that place too...

snowell commented on stardog-spring-batch/build.gradle:
```diff
@@ -85,58 +85,62 @@ signing {
+ 				name 'stardog-spring-batch'
+ 				packaging 'jar'
+ 				description 'Stardog Spring bindings for using Stardog with the Springframework Batch subproject'
+ 				url 'https://github.com/clarkparsia/stardog-spring/stardog-spring-batch'
```
Is this really still under the clarkparsia org? Do we want to migrate it?

snowell replied:
Actually it's obviously in stardog-union/stardog-spring now. So we should probably change it here

snowell commented on stardog-spring-batch/build.gradle:
```diff
@@ -85,58 +85,62 @@ signing {
 					developer {
-						id 'gcoluni'
-						name 'Gregory Coluni'
+						id 'stardog_support'
```
I would either put your name or something like stardog_oss, since "support" probably won't always maintain it

gcoluni replied:
I think that is the user Noah just created. Not sure if that would require all the jira ticket nonsense again.

noahgorstein replied:
Yeah the user in Jira I created is `stardog_support`. I. think it has to be this unless I create a new account which isn't the end of the world.

snowell commented on stardog-spring/build.gradle:
```diff
@@ -32,8 +33,8 @@ dependencies {
-	compile ('com.complexible.stardog:client-http:7.2.1')
- 	testCompile ('com.complexible.stardog:server:7.2.1') {
+	compile ('com.complexible.stardog:client-http:7.3.2')
+ 	testCompile ('com.complexible.stardog:server:7.3.2') {
```
Same Stardog version comment as above

snowell commented on stardog-spring/build.gradle:
```diff
@@ -88,59 +89,63 @@ signing {
 					developer {
-						id 'gcoluni'
-						name 'Gregory Coluni'
+						id 'stardog_support'
```
Same stardog_oss comment as above

gcoluni replied:
Same reply as above.

noahgorstein commented:
Addressed PR comments and modified build.gradle s. I did a test deploy just to the staging repository (not actually released). See the jars here: https://oss.sonatype.org/#stagingRepositories. 

Alternatively, here's a screenshot:
![Screen Shot 2020-07-09 at 2 03 48 PM](https://user-images.githubusercontent.com/23270779/87074543-0562ea80-c1ed-11ea-95c0-77307ffd9b90.png)
